### ISSUE TITLE
[Question] Plane folder icon became a... car

### ISSUE THREAD

**Issue by RememberTheAir** [telegramdesktop/tdesktop#24124]
Is this expected?

![image](https://user-images.githubusercontent.com/16143208/156127324-f82279b3-07b9-4393-8423-6f81fe36c1ec.png)

**Comment by Aokromes** [comment]
yeah noticed it too, i used the plane icon for telegram based groups xd

**Comment by 23rd** [comment]
Beep-beep.
